### PR TITLE
Feature/make new cover button

### PR DESCRIPTION
**Pull request by elizhahn**
## Is this a fix or a feature?
feature
## What is the change?
- When user clicks the View Saved Covers button, the saved covers view is displayed.
- When user clicks the Home button, the home view is displayed, and the Save cover button and Show New Random Cover button is displayed. 
- When home view is displayed, the home button is hidden. 
## What does it fix?
no fix
## Where should the reviewer start?
lines 54-67 implement these new behaviors
## How should this be tested?
- Click on Make Your Own Cover button: form view should be displayed
- Click on View saved covers button: new empty display is shown
- Click on Home button: home view is displayed, proper buttons displayed

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,33 +1,67 @@
...
+var homeView = document.querySelector(".home-view");
+var savedView = document.querySelector(".saved-view");
+var formView = document.querySelector(".form-view");
 
 
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 
+window.addEventListener("load", createRandomCover);
```
Comment (patrickwhardy) [inline src/main.js:26]: could be good to query all your elements and bind all your click handlers after load. could be something like window.addEventListener("load", initialize). Then
```
function initialize() {
  query all dom elements
  add all event listeners
  createRandomCover()
}
```

Maybe the bug you were running into is a race condition where sometimes the page was loaded by the time you queried "Make new cover" button, and sometimes the JS executed before the button had been added to the DOM.

File: src/main.js [hunk 2]
```diff
@@ -1,33 +1,67 @@
...
-randomCoverButton.addEventListener('click', createRandomCover);
+//when user hits home, it takes them to the home page
+function showHome() {
+  savedView.classList.add("hidden");
+  formView.classList.add("hidden");
+  btnHome.classList.add("hidden");
+  homeView.classList.remove("hidden");
+  btnSaveCover.classList.remove("hidden");
+  btnRandomCover.classList.remove("hidden");
+}
```
Comment (patrickwhardy) [inline src/main.js:78]: can you think of a way to make showForm, showSaved and showHome less repetitive? maybe you can group these elements into what screen they are a part of

**Review by patrickwhardy — APPROVED** [review]
nice work!

File: src/main.js [hunk 3]
```diff
@@ -23,7 +28,23 @@ btnMakeNewCover.addEventListener("click", showForm);
...
+  btnRandomCover.classList.add("hidden");
+  btnHome.classList.remove("hidden");
+}
+
+function returnHome() {
+  homeView.classList.remove("hidden");
+  btnSaveCover.classList.remove("hidden");
+  btnRandomCover.classList.remove("hidden");
+  btnHome.classList.add("hidden");
+}
```
Comment (elizhahn) [inline src/main.js:47]: Refactored Functions for DRY code.

File: src/main.js (hunk 2 above)
Comment (elizhahn) [reply src/main.js:78]: We pushed up a change with some refactored functions. Thanks for the review!

**Pull request merged**